### ISSUE TITLE
WEBUI Autoscroll preset : wrong scroll position [BUG]

### ISSUE THREAD

**Issue by majjam** [nonlinear-labs-dev/C15#3105]
if preset is selected by mouse click WEBUI centers view to previous selected preset - but not the new one.
if preset is selected via HW buttons  WEBUI centers view to correct position.

tested in Chrome and Librewolf (Firefox)

**Comment by schlenkibus** [comment]
I was unable to reproduce the issue
@majjam on `main` the current behavior for selecting an preset via a second WebUI looks like this: 

https://user-images.githubusercontent.com/15381257/164429140-eb6f37f5-b7f1-43fd-9405-16b532f84d80.mp4

Do you have more detail on how I could reproduce this issue?

**Comment by majjam** [comment]
Hi here is what i do: (all with mouse)
1. select preset
2. navigate to a bank outside of the current view so that current preset is not seen.
3. select a new preset in current view
=> center position is wrong

https://user-images.githubusercontent.com/4758563/164493023-16ec327f-2816-4714-9a5c-c768b2a23810.mp4



![grafik](https://user-images.githubusercontent.com/4758563/164492179-305cb59d-ca34-45e7-a885-9e603ee1bc7d.png)

**Comment by majjam** [comment]
@schlenkibus Can you work with that ? or do you need more info ?

**Comment by schlenkibus** [comment]
I will see what I can do, thanks for now! If I need further information I will note that here.

**Comment by MatthiasSeeber** [comment]
This issue should be fixed since 2022-04-27, see #3131.

**Comment by majjam** [comment]
here at milestone-maintenance with 7500 commits the problem stays
Shall we reopen it ? or make anew one ?

**Comment by MatthiasSeeber** [comment]
Indeed. Clicking through Presets in the main WebUI area behaves buggy, as shown above.

If the Preset to be loaded was the selected Preset in the Bank, there is no Bug, otherwise it appears.

Stepping through Presets in the Sound Tab seems to work. Stepping through Banks in the Sound Tab doesn't trigger any scrolling.

**Comment by majjam** [comment]
for me it seems to be more intense if the webui is zoomed
